smsdk/smsdk_entities/raw_data/raw_data.py

```diff
@@ -0,0 +1,49 @@
+import json
+from typing import List
+
+try:
+    import importlib.resources as pkg_resources
+except ImportError:
+    # Try backported to PY<37 `importlib_resources`.
+    import importlib_resources as pkg_resources
```

Comment:
FYI, PY3.7 is no longer a valid release.
- https://devguide.python.org/versions/